sdk/azidentity/azidentity.go

```diff
@@ -209,3 +209,10 @@ func validTenantID(tenantID string) bool {
 	}
 	return match
 }
+
+func getIdentityEndpointSuffix(tenantID string) string {
+	if tenantID == "adfs" {
```

Comment:
It's really this simple?

Reply:
Yes, seems so. The only thing left to do is validate all of the credentials against an in-house ADFS instance. I've talked to someone on the identity crew, so I'll do that before moving forward with this PR.

Reply:
Do we need to add documentation for this behavior?  Seems hidden at present.

Reply:
Do we need to add documentation for this behavior?  Seems hidden.